tests/periph_rtt/main.c

```diff
@@ -47,11 +49,28 @@ void cb(void *arg)
 int main(void)
 {
     puts("\nRIOT RTT low-level driver test");
-    puts("This test will display 'Hello' every 5 seconds\n");
+
+    puts("RTT configuration:");
+    printf("RTT_MAX_VALUE: 0x%08x\n", (unsigned)RTT_MAX_VALUE);
+    printf("RTT_FREQUENCY: %u\n\n", (unsigned)RTT_FREQUENCY);
+
+    puts("Testing the tick conversion");
+    for (unsigned i = 0; i < ARRAY_SIZE(_ticktest); i++) {
+        uint32_t sec = RTT_TICKS_TO_SEC(_ticktest[i]);
+        uint32_t ticks = RTT_SEC_TO_TICKS(sec);
+        printf("Trying to convert %u to seconds and back\n", (unsigned)_ticktest[i]);
```

Comment:
```suggestion
        printf("Trying to convert %" PRIu32 " to seconds and back\n", _ticktest[i]);
```
and maybe also for line 55 ?
(I think this test should also work on 8bit AVR)